stories/index.tsx

```diff
@@ -2,16 +2,18 @@ import styleProcessor from 'wix-style-processor';
 
 styleProcessor.init();
 
-require('../src/components/Autocomplete/docs/index.story');
-require('../src/components/Button/docs/index.story');
-require('../src/components/StatesButton/docs/index.story');
-require('../src/components/Card/docs/index.story');
-require('../src/components/Divider/docs/index.story');
-require('../src/components/Input/docs/index.story');
-require('../src/components/OverlappingCard/docs/index.story');
-require('../src/components/StripCard/docs/index.story');
-require('../src/components/Tabs/docs/index.story');
-require('../src/components/Text/docs/index.story');
-require('../src/components/Grid/docs/index.story');
-require('../src/components/ToggleSwitch/docs'); // TODO: convert to .story
-require('../src/components/Tooltip/docs/index.story');
+import '../src/components/Autocomplete/docs/index.story';
```

Comment:
This should be written with `require` at the moment, because the storybook won't work otherwise.
It's because of some changes I made yesterday.
We had a small problem after your `svg` pull request (not because of something you did), where the icon assets weren't exported to users.
So we made a bit of a change in the project, and it required us to use `require` in this file.